src/backend/looped/session/block_ceo_creation.cpp

```diff
@@ -0,0 +1,37 @@
+#include "backend/looped_command.hpp"
+#include "natives.hpp"
+#include "services/players/player_service.hpp"
+#include "services/script_patcher/script_patcher_service.hpp"
+#include "pointers.hpp"
+#include "core/scr_globals.hpp"
+
+namespace big
+{
+	class block_ceo_creation : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_enable() override
+		{
+			g_script_patcher_service->update();
+		}
+
+		virtual void on_tick() override
+		{
+			if (*g_pointers->m_gta.m_is_session_started && !STREAMING::IS_PLAYER_SWITCH_IN_PROGRESS() && SCRIPT::GET_NUMBER_OF_THREADS_RUNNING_THE_SCRIPT_WITH_THIS_HASH("maintransition"_J) == 0)
```

Comment:
Player Switch In Progress is a candidate for the unlikely suggestion to the compiler. Break these out if you agree.